src/prebid.js

```diff
@@ -603,6 +603,18 @@ $$PREBID_GLOBAL$$.getHighestCpmBids = function (adUnitCode) {
     .map(removeRequestId);
 };
 
+/**
+ * Mark the winning bid as used, should only be used in conjunction with video
+ * @param {string} adUnitCode - required ad unit code
+ * @alias module:pbjs.markWinningBidAsUsed
+ */
+$$PREBID_GLOBAL$$.markWinningBidAsUsed = function (adUnitCode) {
```

Comment:
@Caspervw 
Can you make this accept an object instead of a string?
I'm thinking we would also want to match against a bidId as well as a adUnitCode.
example:
```
{
  adUnitCode : <code>,
  bidId : <id>
}
```

Reply:
Sure thing